code/modules/clothing/suits/wintercoat.dm

```diff
@@ -211,6 +211,11 @@
 		/obj/item/weapon/reagent_containers/food/drinks/shaker,
 		/obj/item/weapon/reagent_containers/food/drinks/discount_shaker)
 
+/obj/item/clothing/suit/storage/wintercoat/druid
```

Comment:
did you test this? the hood is a separate item you know, and that icon looks like it includes both

I believe by default this will receive the hood up/hood down action button

Reply:
Yes, it's tested and the toggle works fine.